README.md

```diff
@@ -1,8 +1,307 @@
+```bash
+$ ./bin/ipc-infra/run-subnet-docker.sh <lotus-api-port> <validator-libp2p-port> <subnet-id> <absolute-path-validator-key>
+
+# Sample execution
+$ ./bin/ipc-infra/run-subnet-docker.sh 1239 1349 /root/t01002 ~/.ipc-agent/wallet.key
+```
+> Beware: This script doesn't support the use of relative paths for the wallet path.
+
+The end of the log of the execution of this script provides a bit more of information than the previous one as it is implemented to be used for production deployments:
+```bash
+>>> Subnet /root/t01002 daemon running in container: 22312347b743f1e95e50a31c1f47736580c9a84819f41cb4ed3d80161a0d750f
+>>> Token to /root/t01002 daemon: eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJBbGxvdyI6WyJyZWFkIiwid3JpdGUiLCJzaWduIiwiYWRtaW4iXX0.TnoDqZJ1fqdkr_oCHFEXvdwU6kYR7Va_ALyEuoPnksA
+>>> Default wallet: t1cp4q4lqsdhob23ysywffg2tvbmar5cshia4rweq
+>>> Subnet subnet validator info:
+t1cp4q4lqsdhob23ysywffg2tvbmar5cshia4rweq@/ip4/172.17.0.3/udp/1348/quic/p2p/12D3KooWN5hbWkCxwvrX9xYxMwFbWm2Jpa1o4qhwifmS t1cp4q4lqsdhob23ysywffg2tvbmar5cshia4rweq@/ip4/127.0.0.1/udp/1348/quic/p2p/12D3KooWN5hbWkCxwvrX9xYxMwFbWm2Jpa1o4qhwifmS t1cp4q4lqsdhob23ysywffg2tvbmar5cshia4rweq@/ip4/172.17.0.3/tcp/1347/p2p/12D3KooWN5hbWkCxwvrX9xYxMwFbWm2Jpa1o4qhwifmSw3Fb t1cp4q4lqsdhob23ysywffg2tvbmar5cshia4rweq@/ip4/127.0.0.1/tcp/1347/p2p/12D3KooWN5hbWkCxwvrX9xYxMwFbWm2Jpa1o4qhwifmSw3FbaVcL
+>>> API listening in host port 1239
+>>> Validator listening in host port 1349
+```
+> Beware: The validator address specified here should be the same as the one that will be used in the next step to join the subnet.
+
+This log provides information about the API and auth tokens for the daemon, default validator wallet used, the multiaddresses where the validator is listening, etc. To configure our IPC agent with this subnet daemon, we need to once again update our IPC agent with the relevant information. In this case, for the sample execution above we need to add the following section to the end of our config file:
+```toml
+[subnets."/root/t01002"]
+id = "/root/t01002"
+jsonrpc_api_http = "http://127.0.0.1:1239/rpc/v1"
+jsonrpc_api_ws = "wss://example.org/rpc/v0"
+auth_token = "eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJBbGxvdyI6WyJyZWFkIiwid3JpdGUiLCJzaWduIiwiYWRtaW4iXX0.TnoDqZJ1fqdkr_oCHFEXvdwU6kYR7Va_ALyEuoPnksA"
+accounts = ["t1cp4q4lqsdhob23ysywffg2tvbmar5cshia4rweq"]
+```
+As always, remember to run `./bin/ipc_agent reload-config` for changes in the config of the agent to be picked up by the daemon.
+
+#### Joining a subnet
+With the daemon for the subnet deployed, we can join the subnet:
+```bash
+$ ./bin/ipc_agent join-subnet --subnet=<subnet-id> --collateral=<collateral_amount> --validator-net-addr=<libp2p-add-validator>
+
+# Sample execution
+$ ./bin/ipc_agent join-subnet --subnet=/root/t01002 --collateral=2 --validator-net-addr="GET_ADDRESS_FROM_SCRIPT"
```

Comment:
I realise I was the one who suggested replacing the example address with a placeholder, but now I realise it isn't obvious how to pass the address, given the deploy script outputs 4 addresses and you seem to want to substitute `/dns/host.docker.internal/` as the address (further below, section on changing address)?